docs/modules/ROOT/nav.adoc

```diff
@@ -0,0 +1,2 @@
+* xref:index.adoc[Overview]
```

Comment:
Are we using "Overview" or "Getting started"?

Reply:
Overview is the landing, with description, sample usage, and links.

Getting Started is an in-depth guide to using the thing. We don't have one for some of our packages (like test-helpers, which is is just overview, config and API ref).

I'll put together some guidelines to keep everything consistent.